app/views/events/_form.html.erb

```diff
@@ -11,27 +11,31 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name %>
+  <div class="form-group">
+    <label for="name"> <%= t('.name')%> </label>
```

Comment:
As I said in previous PR this will not work. You need to add bootstrap classes into rails/ruby code not remove it.

Reply:
still relevant